public/pages/CreateRollup/containers/CreateRollupForm/CreateRollupForm.tsx

```diff
@@ -143,6 +150,59 @@ export default class CreateRollupForm extends Component<CreateRollupFormProps, C
     this._isMount = false;
   }
 
+  componentDidUpdate(prevProps: CreateRollupFormProps, prevState: Readonly<CreateRollupFormState>) {
+    if (prevProps.dataSourceId != this.props.dataSourceId) {
```

Comment:
Please refactor this to a common function.

Reply:
Same as createPolicy